zio-http/src/main/scala/zio/http/socket/package.scala

```diff
@@ -21,10 +21,10 @@ package object socket {
   /**
    * A channel that allows websocket frames to be written to it.
    */
-  type WebSocketChannel = Channel[WebSocketFrame]
+  type WebSocketChannel = Channel[WebSocketChannelEvent, WebSocketChannelEvent]
```

Comment:
Possibly we could move these two type aliases to `zio.http` and delete the whole `.socket` package? It seems to thin to keep.